### ISSUE TITLE
Tooltip: when icon is used as the element which provides tooltips it's missing from the accessibility tree

### ISSUE THREAD

**Issue by KKoukiou** [patternfly/patternfly-react#5535]
See the example from the documentation here: https://www.patternfly.org/v4/components/tooltip/#on-icon
The SVG has aria-hidden="true", which causes to be missing from the accessibility tree. However, when users use icons for triggering tooltips I guess they expect also users which use assistive technologies to be able to read the tooltip content.

What it the right approach in this case? Could we fix the example accordingly?

**Comment by jessiehuff** [comment]
Hey @KKoukiou! 🙂  I'm curious if you're using this in product right now or if you just noticed the issue in the example. And if you are using it in product, what's the use case? 
You definitely point out a valid issue in our example, and it's one that Jenn and I have debated for awhile. It's a lot easier when you have an interactive element that you're putting a tooltip on because focus already goes to the item so the tooltip can open on focus. However, in non-interactive elements, it's trickier because you not only have to manage the focus for keyboard users but also find a way to make sure that's communicated by screen reader as well, especially in a situation like this where we typically have the svg itself hidden to screen readers. Most examples I've seen with tooltips are on interactive elements like buttons, and I haven't seen an article describing non-interactive elements having tooltips yet. I have an idea of how we could make this example accessible, but I'll keep looking for other suggestions in case there's a standard for this.

**Comment by KKoukiou** [comment]
We actually have some usages of tooltips on plain icons in cockpit. See screenshot [1].
We can of course change the approach on out side, since it's already discussed and there is not a better solution. 

[1] Here are software updates page from cockpit. We use icons to show the severity of a pending update. Tooltip with the explanatory text.

![Screenshot from 2021-03-22 21-46-12](https://user-images.githubusercontent.com/14921356/112056312-25a6bf80-8b58-11eb-931d-f2b52b52127f.png)

**Comment by KKoukiou** [comment]
@jessiehuff I have also no idea for a solution here. Since you already discussed it with Jenn feel free to close the issue.

**Comment by jessiehuff** [comment]
Hey @KKoukiou, I added a PR that updates our PF example, but with your use case I actually don't recommend using a tooltip here and adding all of those tab stops. If we add a tab stop for keyboard users for every tooltip, I could see that creating a decent burden on their navigation. Is there a way to work with design to see if you can provide a legend here instead? That seems better for mouse and keyboard users than creating tooltips on every icon.

I'd like to add a comment to my fix somehow saying it's only recommended when there are few icons on a page, and if there are more than one icon like your use case, to consider using a legend.
